packages/common/src/entities/dbmss/dbmss.local.ts

```diff
@@ -523,7 +549,22 @@ export class LocalDbmss extends DbmssAbstract<LocalEnvironment> {
             throw new AmbiguousTargetError(`DBMS ${dbmsId} not found`);
         }
 
-        await fse.unlink(dbmsRootPath).catch(() => fse.remove(dbmsRootPath));
+        let targetPath: string | undefined;
+        if (dbms.isCustomPathInstallation) {
+            targetPath = await fse.readlink(dbmsRootPath);
+        }
+
+        // unlink / remove DBMS dir from Relate data dir
+        await fse
+            .unlink(dbmsRootPath)
+            .then(() => {
+                // remove target if custom DBMS install path was used
+                if (dbms.isCustomPathInstallation && targetPath) {
+                    return fse.remove(targetPath);
+                }
+                return Promise.resolve();
+            })
+            .catch(() => fse.remove(dbmsRootPath));
```

Comment:
suggestions please

Reply:
Need to check what happens on `fse.readlink` if the symlink is broken (eg. the target path was moved/deleted by the user), and if `fse.unlink` works to delete a broken link (since we're deleting the target path before the symlink), but I figure this could work: 

```suggestion
        // Remove DBMS directory from custom installation path
        if (dbms.isCustomPathInstallation) {
            const targetPath = await fse.readlink(dbmsRootPath);
            await fse.remove(targetPath);
        }

        // Remove DBMS directory or symlink from Relate data dir
        await fse.unlink(dbmsRootPath).catch(() => fse.remove(dbmsRootPath));
```

Reply:
Ill probably do it this. I dont think we would even necessarily get to the `readLink` if symlink is broken given `!found` above this block. But I will double check